File: python/java_to_python/07-pgmThread/src/co/edu/campusucc/poo/MainThreadRunnable.py
import random
import threading
from ThreadRunnable import ThreadRunnable

def main():
    runnable = ThreadRunnable()
    runnable.setTimes(random.randint(0, 50))
    # initializing Thread Object
    thread = threading.Thread(target=runnable.run)
    thread.start()

if __name__ == "__main__":
    main()

File: ruby/pgms_ruby/07-pgmThread/MainThreadRunnable.rb
require 'securerandom'
require 'thread'
require_relative 'ThreadRunnable'

def main
  runnable = ThreadRunnable.new
  runnable.setTimes(SecureRandom.random_number(50))
  # initializing Thread Object
  thread = Thread.new { runnable.run }
  thread.start
end

if __FILE__ == $0
  main
end